stories/black-belt-climate-ej.stories.mdx

```diff
@@ -296,10 +266,9 @@ taxonomy:
      * [Census Bureau Data | Occupation](https://www.census.gov/topics/employment/industry-occupation.html)]
     </p>
     <p>
-     **Editor**: Maheshwari Neelam and Derek Koehl; 
-     **Developers**: Andrew Blackford, Trent Cowan, Jerika Christman, Brian Freitag, and Aaron Kaulfus ;
+     **Editors**: Maheshwari Neelam and Derek Koehl; 
+     **Developers**: Andrew Blackford, Trent Cowan, Jerika Christman, Brian Freitag, and Aaron Kaulfus;
      **Science and Content Contributors**: Maheshwari Neelam and Udaysankar Nair;
```

Comment:
```suggestion
     **Editors**: Maheshwari Neelam and Derek Koehl;
     
     **Developers**: Andrew Blackford, Trent Cowan, Jerika Christman, Brian Freitag, and Aaron Kaulfus;
     
     **Science and Content Contributors**: Maheshwari Neelam and Udaysankar Nair;
```

if you want them to be on separate lines (which I think will look better), do this ☝️